src/index.js

```diff
@@ -6,6 +6,8 @@ const { pushView } = require('./utils')
 
 module.exports = async function (req, res) {
   const { pathname, query } = url.parse(req.url, /* parseQueryString */ true)
+  res.setHeader('Access-Control-Allow-Origin', '*')
+  res.setHeader("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept")
```

Comment:
Ok so I tested and found that no headers are actually necessary so I'm reverting last commit.